test/lux/invalidate_default/run.lux

```diff
@@ -0,0 +1,12 @@
+[doc Test of YANG model deviation compilation and use]
+
+[config require=MAKE]
+
+[global fail_pattern=[Ee][Rr][Rr][Oo][Rr]]
+
+################################################################################
+
+[shell test]
+    -type
+    !~/proj/yanger/bin/yanger -f sn deviated.yang --deviation-module dev.yang | grep -A1 -E 'default.*any'
```

Comment:
Please do s;~/proj/yanger/bin/yanger;yanger;

Also, does this test really fail before your fix?

Reply:
Yes, before the fix it fails, since there is {type, enumeration...} in the printout (as part of the sn.default tuple).